Objects/listobject.c

```diff
@@ -120,25 +121,24 @@ list_preallocate_exact(PyListObject *self, Py_ssize_t size)
 }
 
 void
-_PyList_ClearFreeList(PyInterpreterState *interp)
+_PyList_ClearFreeList(_PyFreeListState *state)
 {
 #if PyList_MAXFREELIST > 0
-    struct _Py_list_state *state = &interp->list;
-    while (state->numfree) {
-        PyListObject *op = state->free_list[--state->numfree];
+    struct _Py_list_state *list_state = &state->list;
+    while (list_state->numfree) {
+        PyListObject *op = list_state->free_list[--list_state->numfree];
```

Comment:
You can get a smaller and more readable diff by renaming the function argument (for example to `freelist_state`, instead of renaming the local variable ;)

```suggestion
    struct _Py_list_state *state = &freelist_state->list;
    while (state->numfree) {
        PyListObject *op = state->free_list[--state->numfree];
```

After all, it is the _function input_ that is changed; not the loop body.

Reply:
Cool! Applied!